### DISCUSSION TITLE
Custom report - returns no matches when trying to modify

### DISCUSSION THREAD

**Discussion by rob-bc** [GothenburgBitFactory/taskwarrior#2725] [Q&A] [answered]
I have defined a custom report:
```
report.month.columns    id,start.age,entry.age,depends,project,tags,recur,scheduled.countdown,due,until.remaining,importance,urg,description,urgency
report.month.context    1
report.month.dateformat h:N a d b
report.month.filter     status:pending +month
report.month.labels     ID,Active,Age,Deps,Project,Tag,Recur,S,Due,Until,Imp.,Urg,Description,Urg
report.month.sort       urgency-,priority-
```

When I run just this report, it returns 24 tasks:
```

$ task month

ID Age       Project  Tag                Recur Due              Imp. Urg Description                                     
80 11d       link     month week               12:00 Wed 26 Jan 5    4   uplink power workshop actions
[...]
```

But when I try to perform a bulk modification to the tasks in that report, I get "no matches":

```
$ task month modify +week
No matches.
```

Does anyone know - is this user error, or a known bug?

**Comment by tbabej** [comment, marked as answer]
Hi @rob-bc,

filtering by report names as in the example you suggest is not currently possible (as far as I know, this was never supported :crying_cat_face:). You, can, however, simply duplicate the report's filter (which is in this case relatively easy):

```
$ task status:pending +month modify +week
```

This can be even done dynamically, one can use `_get` command together with the DOM model in order to get the flter dynamically:

```
$ task `task _get rc.report.month.filter` modify +week
```

though that might not be convenient for the direct use on the CLI (but could be useful in scripts). That said, I think what you're looking for could be best served by the context feature. Defining a context with a filter `status:pending +month`, you can then activate the context and a subsequent

```
$ task modify +week
```

would only affect the tasks that match `status:pending +month`. This is a good fit if, in particular, you find yourself operating on the restricted tasks that match the `+month` tag for a couple of operations in a row.

Hope this helps!
